pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
     "setuptools >= 61",
-    'tomli; python_version < "3.11"',
+    'tomli; python_version <= "3.12"',
```

Comment:
this doesn't need changing, since [tomllib](https://docs.python.org/3/library/tomllib.html) is used in [versioneer.py](https://github.com/pypest/pyemu/blob/d390d5d175ca831abbc6480da8d078107e0f4418/versioneer.py#L37) for py311+

Reply:
ah that's right, good catch @mwtoews